pkg/synthetictests/duplicated_events.go

```diff
@@ -69,7 +66,11 @@ var allowedRepeatedEventPatterns = []*regexp.Regexp{
 	regexp.MustCompile(`ns/e2e-persistent-local-volumes-test-[0-9]+ pod/pod-[a-z0-9.-]+ reason/FailedScheduling`),
 
 	// various DeploymentConfig tests trigger this by canceling multiple rollouts
-	regexp.MustCompile(`.*reason/DeploymentAwaitingCancellation Deployment of version [0-9].+ awaiting cancellation of older running deployments`),
+	regexp.MustCompile(`reason/DeploymentAwaitingCancellation Deployment of version [0-9].+ awaiting cancellation of older running deployments`),
+
+	// Can be caused by registry hiccups and not explicitly tied to any particular component failure
+	// (if image pulls fail catastrophically we'd expect them to lead to test failures)
```

Comment:
I think we want to catch the non-catastrophic ones too.  Consistent lack of availability seems like a problem we want to find.

I'm more flexible for e2e namespaces if you restrict it to ns/e2e-something *** Back-off pulling image

Reply:
So the consistent OLM pull failures seem to have gone away (must have been a registry/image issue).

but we're still at 0.13% of all failures being caused by imagepull backoff event repetition:
https://search.ci.openshift.org/?search=reason%2FBackOff+Back-off+pulling+image&maxAge=48h&context=1&type=junit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

I guess i'm trying to figure out how we draw the line between "this might be useful/good to know about" and "this is causing CI jobs to fail that otherwise would have succeeded and no one is going to investigate this"

whitelisting "e2e-*" as a namespace would probably cover most of them, not clear if you're amenable to that or not.